packages/shared/src/components/CreateFeedFilterButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement, useContext } from 'react';
+import { useQueryClient } from 'react-query';
+import AuthContext from '../contexts/AuthContext';
+import AnalyticsContext from '../contexts/AnalyticsContext';
+import {
+  getFeedSettingsQueryKey,
+  updateLocalFeedSettings,
+} from '../hooks/useFeedSettings';
+import { AllTagCategoriesData } from '../graphql/feedSettings';
+import { Button, ButtonProps } from './buttons/Button';
+
+export default function CreateFeedFilterButton(
+  props: ButtonProps<'button'>,
+): ReactElement {
+  const { user, showLogin } = useContext(AuthContext);
+  const { trackEvent } = useContext(AnalyticsContext);
+  const client = useQueryClient();
+
+  const onCreate = () => {
+    trackEvent({
+      event_name: 'click',
+      target_type: 'create feed filters',
+      target_id: 'feed-filters',
+    });
```

Comment:
We might want to check with @idoshamun  if we want to get the specific version for which this button was clicked.

Reply:
Yes, you are right. We want to set a different `target_id` to every option so we can select the best performing option based on conversion rate

Reply:
I made the `target_id` dynamic based on the version, so we have a different one for each type. Could you please have a look and approve if it makes sense for you? @idoshamun